src/Microsoft.DotNet.Wpf/tests/UnitTests/Directory.Build.targets

```diff
@@ -0,0 +1,36 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
+  <ItemGroup>
+    <PackageReference Include="runtime.$(WpfRuntimeIdentifier).$(DncEngTransportPackageName)"
+                      Condition="'$(RepositoryName)'=='wpf'"
+                      Version="$(MicrosoftDotNetWpfDncEngVersion)">
+      <CopyLocal>true</CopyLocal>
+      <GeneratePathProperty>true</GeneratePathProperty>
+    </PackageReference>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <_DncEngPackagePath Condition="'$(WpfRuntimeIdentifier)'=='win-x64' And '$(Configuration)' != 'Debug'">$(Pkgruntime_win-x64_Microsoft_DotNet_Wpf_DncEng)</_DncEngPackagePath>
+    <_DncEngPackagePath Condition="'$(WpfRuntimeIdentifier)'=='win-x64' And '$(Configuration)' == 'Debug'">$(Pkgruntime_win-x64_Microsoft_DotNet_Wpf_DncEng_Debug)</_DncEngPackagePath>
+    <_DncEngPackagePath Condition="'$(WpfRuntimeIdentifier)'=='win-x86' And '$(Configuration)' != 'Debug'">$(Pkgruntime_win-x86_Microsoft_DotNet_Wpf_DncEng)</_DncEngPackagePath>
+    <_DncEngPackagePath Condition="'$(WpfRuntimeIdentifier)'=='win-x86' And '$(Configuration)' == 'Debug'">$(Pkgruntime_win-x86_Microsoft_DotNet_Wpf_DncEng_Debug)</_DncEngPackagePath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- These exist to ensure that dependencies (esp. native ones) are binplaced with tests correctly -->
+    <None Include="$(_DncEngPackagePath)\runtimes\$(WpfRuntimeIdentifier)\native\*.dll"
+          CopyToOutputDirectory="PreserveNewest"
+          Visible="False" />
+  </ItemGroup>
+
+  <Target Name="RemoveWindowsBaseNetCoreAppReference"
```

Comment:
This section is needed for projects that reference WindowsBase (It removes the facade WindowsBase that exists in the BCL that conflicts with the WPF version of WindowsBase).